communication/slack-config/sig-network/config.yaml

```
@@ -1,3 +1,3 @@
 channels:
-  - name: sig-network-service-apis
+  - name: sig-network-gateway-api
```

```suggestion
  - name: sig-network-gateway-api
    id: CR0H13KGA
```

When renaming a channel, the underlying channel ID is needed 👍

---

Updated, thanks!